project/Dependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt._
 
 object Dependencies {
   lazy val lenthallV = "0.21-e1b7822-SNAP"
-  lazy val wdl4sV = "0.10-cc58e9b-SNAP"
+  lazy val wdl4sV = "0.10-dce653c-SNAP"
```

Comment:
If you do merge this, when you rebase, please keep this at the latest wdl4s to include my changes- `9ad47e0`